src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -20,12 +24,18 @@
 
 @RestController
 @RequestMapping("/api/books")
+@Api
 public class BookController {
 
     @Autowired
     private BookRepository bookRepository;
 
     @GetMapping("/{author}")
+    @ApiOperation(value = "Given an author returns the book", response = Book.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, response = Book.class, message = ""),
```

Comment:
If you want to leave the message empty you shouldn't add the modifier `message`, simply use 
`@ApiResponse(code = 200, response = Book.class)`
However, I do recommend adding a short description of the response 🙂

Reply:
When I remove the message parameter it get marked as an error and would not compile :(